### ISSUE TITLE
Gazebo_ros spawn_model reference_frame not working

### ISSUE THREAD

**Issue by jvgomez** [ros-simulation/gazebo_ros_pkgs#411]
This is a copy and paste from [this question](http://answers.ros.org/question/229879/gazebo_ros-spawn_model-reference_frame-not-working/) ROSAnswers.

I am launching the following world with the `empty_world.launch` file:

```
<include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name"   value="$(find my_pkg)/world.world" />
</include>
```

Where `world.world` is:

```
<sdf version="1.5">
  <world name="default">
    <include>
      <uri>model://sun</uri>
    </include>
    <include>
      <uri>model://ground_plane</uri>
    </include>


    <!-- Extra objects -->
    <model name='unit_cylinder_1'>
        <pose>0 0 2 3.141592 0 1.57</pose>
    ...
```

I am also launching the following:

```
<node ns="alpha" name="spawn_robot" pkg="gazebo_ros" type="spawn_model" output="screen"
      args="-param robot_description
            -urdf -model alpha
            -robot_namespace alpha
            -reference_frame unit_cylinder_1
            -x 2
            -y 4
            -z 0"/>
```

But I keep getting this error: `Spawn status: SpawnModel: reference reference_frame not found, did you forget to scope the link by model name?` Parameter robot_description is in the alpha namespace and it is a valid URDF (it works if I remove the `-reference_frame`).

Basically, my question is: is there a bug on the 1-reference_frame` argument or is there something else I am missing? Maybe is related with namespaces or something.

Thanks!
